mars/metric/backends/ray/ray_metric.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Dict
+
+from ....utils import lazy_import
+from ..metric import (
+    AbstractMetric,
+    AbstractCounter,
+    AbstractGauge,
+    AbstractMeter,
+    AbstractHistogram,
+)
+
+ray_metrics = lazy_import("ray.util.metrics")
+
+# Note: Gauge `record` method is deprecated in ray 1.3.0 version, so here
+# make it compatible with the old and new ray versions.
+RAY_GAUGE_SET_AVAILABLE = (
+    True if ray_metrics and hasattr(ray_metrics.Gauge, "set") else False
+)
+
+
+class RayMetricMixin(AbstractMetric):
+    def _init(self):
+        if ray_metrics:
```

Comment:
Append a `  # pragma: no branch` at the end of the line to avoid [branch coverage report](https://app.codecov.io/gh/mars-project/mars/compare/2749/diff). What's more, it is recommended to use `ray_metrics is None`.

Reply:
Ok. Thanks!